docs/markup-language/README_es.md

```diff
@@ -0,0 +1,1618 @@
+
+```
+^ La Empresa declara y asegura que:
+
+^^(Organización) **Organización**. La Empresa es una sociedad limitada debidamente
+constitutida, con existencia válida, y vigente conforme con las leyes del [[Estado
+de Constitución]], tiene el poder social para llevar a cabo los negocios según se
+hace actualmente, y reúne los requisitos para hacer negocios en toda jurisdicción
+en la cual el tipo y la ubicación de los bienes de los cuales es el dueño, o el
+tipo de negocio que hace exige calificación, o en el cual el no así calificar le
+ocasionaría un efecto material adverso. No hay procesos pendientes, ni al
+conocimiento de la Empresa, se ha amenazado con abrir uno, en el cual se le acusa
+de que el tipo de su negocio hace que la calificación sea necesaria en ninguna otra
+jurisdicción.
+
+...
+
+Salvo por disposición en la Sección [[Organización]], ...
+```
+
+## Titles ("Títulos")
+
+Puedes esconder el títlo de un modelo de un acuerdo al incluir la opción siguiente al principio del documento. Esta opción es útil para suprimir un título para cartas o al descargar un documento tipo Word.
+
+```
+####
+show title:false;
+####
+```
+
+## Anotación
+
+A veces puede ser útil incrustar anotaciones en tu modelo para darle contexto al usuario. La anotación se rinde sólo en vista previa y no afecta a la versión docx o pdf del acuerdo.
+
+Existen dos tipos de anotaciones en el lenguaje de marcado actualmente: el membrete y la nota.
+
+La anotación de membrete se coloca por encima del resto del modelo en el editor y se puede usar (por ejemplo) para brindar una descripción del contenido del modelo para el lector. La sintáxis para la anotación de un membrete es lo siguiente:
+
+```
+una disposición
+
+'''
+Esto es una anotación de membrete para explicar la disposicón, y por qué viene bien tenerlo aquí.
+'''
+
+otra disposición
+```
+
+La anotación de notas se puede colapsar dentro del cuerpo del modelo. Se puede usar para añadir notas notas a lo largo de ello para aclarar o comentar sobre el contenido. La sintáxis para la anotación de notas es lo siguiente:
+
+```
+una disposición
+
+"""
+Esto es una anotación de membrete para explicar la disposicón, y por qué viene bien tenerlo aquí.
+"""
+
+otra disposición
+```
+
+## Tablas
+
+Los datos tabulares se muestran fácilmente usando tablas. Una tabla se define con el uso de carácteres de pleca (`|`) como separadores de columna, y rayas (`-`) para separar el membrete de la tabla de las filas de datos.
+
+```
+| Membrete 1 | Membrete 2 | Membrete 3 |
+| ---------- | ---------- | ---------- |
+| Datos 1    | Datos 2    | Datos 3    |
+| {{var1}}   | {{var2}}   | {{var3}}   |
+```
+
+Observa que los separadores de columna no tienen que alinearse de fila a fila, así que lo siguiente también es una definición válida para una tabla.
+
+```
+| H1 | Membrete 2 | H3 |
+| - | - | - |
+| Datos 1 | D2 | D3 |
+```
+
+Las células de una tabla pueden contener contenido en texto, variables, o condicionales.
```

Comment:
Take this with a grain of salt, as my Spanish is obviously much weaker than yours, but is the "contenido" necessary or would `pueden contener` be sufficient?

Reply:
Yes this is a funny one, and my original draft did what you said. However, in that case it would just be saying 'the table can contain text, variables or conditionals', not 'the table can hold content in text, variables, or conditionals'. 

I'll update this to a cognate of 'contener' to retain the meaning of the original but avoid the repetitive-sounding Spanish. Good catch, thanks!

Reply:
Thanks for clarifying. That makes sense and sounds like a good compromise.